modules/querying/pages/syntax-versions.adoc

```diff
@@ -0,0 +1,84 @@
+Therefore, Table 1 specifically lays out the differences in the `FROM` clause.
+Other differences are summarized in Table 2.
+
+.Differences in the `FROM` clause
+|===
+|Feature|V1 |V2
+
+|Step vs path pattern
+a|
+The keyword `FROM` is followed by a 1-hop step.
+Supports 1-hop only.
+
+For more information, see xref:select-statement/select-statement-v1.adoc#_from_clause[`FROM` clause (V1)].
+a|
+The keyword `FROM` is followed by a xref:select-statement/index.adoc#_path_pattern[path pattern].
+The path pattern may consist of multiple hops and allows for powerful xref:tutorials:pattern-matching/multiple-hop-and-accumulation.adoc[pattern matching].
+
+|Edge notation
+|Edges in a step can optionally have a right arrowhead outside the parentheses.
+For example:
+`Start:s -( Edges:e )-> Target:t`
+|No arrowheads are allowed outside the parentheses.
+Each edge may have arrowheads inside the parentheses to show the direction.
+For example:
+`Start:s -(Edge1>:e1)- Middle:m -(<Edge2:e2)- Target:t`
+
+
+|Traversal direction
+|The traversal action is from left to right, and the notation uses either a rightward facing arrowhead or no arrowhead to depict a connection.
+|The traversal direction is under the control of the query writer, with arrowheads on each edge set to show the direction.
+
+|Source vertex set
+|The source vertex set in a step in the `FROM` clause must be xref:declaration-and-assignment-statements.adoc#_vertex_set_variables[a vertex set (seed set)] declared earlier in the query.
+|The xref:select-statement/index.adoc#_source_vertex_set[source vertex set path pattern] in `FROM` clause can be a vertex set, a vertex type, or an accumulator.
+
+|===
+
+.Other differences
+|===
+|Feature |V1 |V2
+
+|xref:select-statement/index.adoc#_post_accum_clause[`POST-ACCUM` clause]
+a|* Each `SELECT` statement can only have 1 `POST-ACCUM` clause.
+* A `POST-ACCUM` clause may have statements that reference both source and target vertex aliases.
+a|
+* Each `SELECT` statement can have multiple `POST-ACCUM` clauses.
+* Each `POST-ACCUM` clause must bind itself with one vertex alias.
+This can be done explicitly or implicitly.
+
+|xref:select-statement/sql-like-select-statement.adoc[SQL-like `SELECT` statement]
+|Not supported.
+|Supported.
+
+|Data modification statements
+|`FROM` clause can only perform one hop, which is the same as in the `SELECT` statement in V1.
+|`FROM` clause can only perform one hop, even though it may perform multiple hops in the `SELECT` statement.
+
+|===
+
+== Changing a query from V1 to V2
+GSQL can detect the syntax version of the query automatically, so there is no need to modify your queries from V1 to V2.
+However,
+
+. Delete arrowheads outside parentheses in the `FROM` clause.
+* This is the most common incompatibility issue between a query written in V1 and V2.
+Most V1 queries can run smoothly in V2 after the arrowheads are deleted.
```

Comment:
"Most V1 queries can run smoothly in V2 after the arrowheads are deleted."

This is not necessarily true since the edge type can be directed. The more accurate way to say is "... after the arrowheads are deleted to moved."

Reply:
Added note about arrowheads